### ISSUE TITLE
Rule only working with deprecated value

### ISSUE THREAD

**Issue by ChrizzDF** [AtomLinter/linter-stylelint#230]
First - thanks for this awesome package! =)

I installed the latest linter-stylelint package v.2.17.0 and I defined the following rules:
```
{
  "rules": {
    "string-quotes": "double",
    "font-family-name-quotes": "always-unless-keyword"
  }
}
```
as described in the 6.5.1 release [here](https://github.com/stylelint/stylelint/releases/tag/6.5.1).

Well... but using the deprecated value does work!
```
{
  "rules": {
    "font-family-name-quotes": "double-unless-keyword"
  }
}
```

Maybe someone can confirm this behaviour or show me what I'm doing wrong.

**Comment by ChrizzDF** [comment]
Ok, I'm getting the following Warning message in Atom when using the `"double-unlesskeyword"` rule:

_The 'double-unless-keyword' option for 'font-family-name-quotes' has been deprecated, and will be removed in '7.0.' Instead, use the 'always-where-required', 'always-where-recommended' or 'always-unless-keyword' options together with the 'string-quotes' rule._

So: the deprecated one works when linting my code but throws a Warning.
The new rule doesn't work, but therefore no issues... 😂

**Comment by Arcanemagus** [comment]
So just to clarify, you are using a deprecated option, which is working, but giving you the above message. When you try to use one of the options it recommends as an alternative it doesn't work though?

Does `stylelint` on the CLI exhibit the same behavior?

**Comment by ChrizzDF** [comment]
Yes, you understood everything totally correct.
Right now I'm only using stylelint in Atom, not in the CLI in my Ember App that I'm developing.

My goal is to have double quotes around each FontFamily name that is no keyword, described [here](http://stylelint.io/user-guide/rules/font-family-name-quotes/)

So my `stylelintrc` file looks like:
```
{
  "rules": {
    "string-quotes": "double",
    "font-family-name-quotes": "always-unless-keyword"
  }
}
```
if I then write the following CSS rule: `font-family: 'Material Icons';` an error occures and when I change it to double quotes, everything is fine, BUT if a write `font-family: Arial;`where Arial is no keyword and the FontFamily name, NO ERROR is showing up, although it's considered a warning according to the official docs I linked above.

Can you still follow?^^

**Comment by Arcanemagus** [comment]
Looks like this is a bug in `stylelint@6.9.0` _but_, it's already fixed in `stylelint@7.0.0` 😉:
![image](https://cloud.githubusercontent.com/assets/427137/16815362/642df4f0-48ee-11e6-9867-7679b21e32be.png)

The update to that version is being done in https://github.com/AtomLinter/linter-stylelint/pull/233, as soon as the CI builds all finish I'll be merging that and releasing it.

**Comment by ChrizzDF** [comment]
Yeah! Thumbs up! Glad you figured it out that quick!
Thank you and please keep going with our nice package! 😎
